src/hotspot/share/gc/shared/collectedHeap.hpp

```diff
@@ -132,6 +132,8 @@ class CollectedHeap : public CHeapObj<mtGC> {
   NOT_PRODUCT(volatile size_t _promotion_failure_alot_count;)
   NOT_PRODUCT(volatile size_t _promotion_failure_alot_gc_number;)
 
+  static jlong _vmthread_cpu_time;
```

Comment:
I wonder if it's more natural to make it non-static.

Reply:
Could you please elaborate? Other variables like `static size_t _stack_chunk_max_size` is also static.